src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -126,6 +127,11 @@ public async Task HandleScriptExecuteRequest(ScriptingParams parameters, Request
                     }
                 }
 
+                if (parameters.FilePath == null)
+                {
+                    parameters.FilePath = Path.Join(Path.GetTempPath(), $"{Convert.ToBase64String(Guid.NewGuid().ToByteArray()).Substring(0, 8)}-{DateTime.Now:yyyy-MM-dd_hh-mm-ss-fff}");
```

Comment:
please add a comment to explain what the path would look like and reason for it.

Reply:
A comment on the code or here?

Reply:
in the code. it's useful to have a sample in a comment, makes it more clear. also may add some comment about why you did that. trying to create a random file name?

Reply:
@aaomidi any reason to not use [GetTempFileName](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=netcore-3.0)?

Reply:
@aaomidi double checking the docs, it looks like [GetRandomFileName](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getrandomfilename?view=netcore-3.0) is the one I'm asking about.  GetTempFileName appears to actually create a file, which isn't what we want I think.